proto/storage/cve.proto

```diff
@@ -200,6 +214,8 @@ message CVSSV2 {
   float impact_score = 9;
   float score = 10;
   Severity severity = 11;
+  Source source = 12; // @gotags: search:"CVSS Source";
```

Comment:
If the same tag is attached to both sources - and both NVD and redhat scores are available - what will the search pick?

Shouldn't we have different tags?

Reply:
Or can we move the `source` out of CVSSV2 and CVSSV3 and add it directly to `message CVSSScore`? Same for `url`.

It will be
```
message CVSSScore{
  Source source // @gotags: search:"CVSS Source";
  string url
  oneof cvss_score{
    CVSSV2 cvssv2 = 1;
    CVSSV3 cvssv3 = 2;
  }
}
```